src/overlays/actors/ovl_En_Dnh/z_en_dnh.h

```diff
@@ -10,7 +10,20 @@ typedef void (*EnDnhActionFunc)(struct EnDnh*, PlayState*);
 typedef struct EnDnh {
     /* 0x000 */ Actor actor;
     /* 0x144 */ EnDnhActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0x74];
+    /* 0x148 */ SkelAnime skelAnime;
+    /* 0x18C */ u16 unk18C;
+    /* 0x18E */ char pad18E[2];
+    /* 0x190 */ s32 unk190;
```

Comment:
```suggestion
    /* 0x18C */ u16 unk18C;
    /* 0x190 */ s32 unk190;
```

s32 will align to 4-byte words, so this padding isn't needed